modules/installation-user-infra-machines-static-network.adoc

```diff
@@ -230,26 +218,38 @@ a|Do not clear partition table on error.
 a|`-h`, `--help`
 a|Print help information.
 
+a| `-f`, `--image-file <path>`
+a|Specify a local image file manually. Used for debugging.
+
+a|
+a|
+
 2+|_Command-line argument_
 
 |*Argument* |*Description*
 
 a|`<device>`
 a|The destination device.
 
+a|
+a|
+
 2+|_coreos-installer embedded Ignition commands_
 
 |*Command* |*Description*
 
-a|`$ coreos-installer iso ignition embed <options> <ISO_image> <Ignition_config>`
+a|`$ coreos-installer iso ignition embed --ignition-file <options> <ISO_image>`
```

Comment:
`--ignition-file` takes a `<file-path>` argument.

Reply:
Do you mean the following?
`$ coreos-installer iso ignition embed --ignition-file <options> <file_path>`

Reply:
Should be ` $ coreos-installer iso ignition embed <options> --ignition-file <file_path> <ISO_image>`.

Reply:
Resolved